mkdocs/pages/en/devbook/accounts/query-balance.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Query Account Balance
+---
+
+# Querying an Account Balance
+
+Accounts on Symbol can hold <mosaics:> (fungible tokens), including the native currency <XYM:>.
+
+This tutorial shows how to query an account's mosaic balances and format them with decimal places and friendly names.
+
+## Prerequisites
+
+This tutorial uses the Symbol REST API without requiring an SDK.
+You only need a way to make HTTP requests.
+
+## Full Code
+
+{% import 'tutorial.jinja2' as tutorial with context %}
+
+{{ tutorial.code_full('devbook/accounts/query-balance', ['py', 'js']) }}
+
+The snippet uses the `NODE_URL` environment variable to configure the Symbol API node.
+If no value is provided, it defaults to `https://001-sai-dual.symboltest.net:3001`.
+
+It defines the following functions:
+
+* `get_account_info()`: Fetches account state by address or public key.
+* `get_mosaic_names()`: Fetches namespace aliases for mosaics.
+* `get_mosaics_info()`: Fetches properties for multiple mosaics in a single request.
+* `format_amount()`: Formats amounts with decimal places.
+
+## Code Explanation
+
+### Fetching Account Information
+
+{{ tutorial.code_snippet(['py:10:23', 'js:6:17']) }}
+
+The <get:/accounts/{accountId}> endpoint retrieves the state of an account, including all the mosaics it holds.
+
+You can query an account using either its <address:> or its <public key:>.
+
+### Fetching Mosaic Names
+
+{{ tutorial.code_snippet(['py:26:50', 'js:19:42']) }}
+
+Mosaics are identified by 64-bit numeric IDs, which can be hard to read and remember.
+To improve usability, mosaics can be linked to human-readable <namespace:> aliases.
+
+The <post:/namespaces/mosaic/names> endpoint accepts multiple mosaic IDs and returns any namespace names that are
+currently linked to them.
+If a mosaic has no linked namespace, it will not appear in the response.
+
+A mosaic can have multiple namespace aliases (different namespaces can link to the same mosaic).
+When this happens, the code displays all aliases separated by commas.
+
+### Fetching Mosaic Properties
+
+{{ tutorial.code_snippet(['py:53:77', 'js:44:67']) }}
+
+To format mosaic balances correctly, the snippet fetches their properties from the network.
+The key property required is <divisibility:>, which defines how many decimal places a mosaic supports.
+
+For example, a mosaic with divisibility 6 stores amounts as integers representing millionths of a unit,
+so an atomic value of `1_000_000` equals `1.000000` when formatted.
+A mosaic with divisibility 0 represents indivisible units (whole numbers only).
```

Comment:
This is a good explanation, but it's already in the Textbook and I don't want to have to maintain two versions.
This is what the <divisibility:> link is for!

You can remove this paragraph.